_install_guide/logging.md

```diff
@@ -133,3 +134,65 @@ docker inspect -f '{{.HostConfig.LogConfig}}' clouddriver
 {% endraw %}```
 
 If the log config isn't what you expect then something is wrong in the Armory Spinnaker config, and if the config is what you expect the problem is likely in your distributed logging setup.
+
+## Monitoring Spinnaker With Datadog
+
+Spinnaker provides a monitoring container which exports metrics from the core sub-services. Below we add two additional containers to our docker-compose setup: `datadog` and `spinnaker-monitoring`.
```

Comment:
what do you think about?
`We'll need to add` two additional containers to our docker-compose....